File: platforms/web/examples/geometry/mesh/add_instance.js
import { Mesh, Instance } from "fragmentcolor";

const m = new Mesh();
const offset = [0.25, 0.10];
const tint = [1.0, 0.0, 0.0, 1.0];
m.addInstance(Instance.new().set("offset", offset).set("tint", tint));

File: platforms/kotlin/examples/geometry/mesh/add_instance.kt
import org.fragmentcolor.*

val m = Mesh()
val offset = listOf(0.25f, 0.10f)
val tint = listOf(1.0f, 0.0f, 0.0f, 1.0f)
m.addInstance(Instance().set("offset", offset).set("tint", tint))